packages/edit-site/src/components/layout/index.js

```diff
@@ -309,10 +308,7 @@ export default function Layout() {
 							{ isEditorPage && (
 								<div
 									className={ classnames(
```

Comment:
The classnames call is useless here as well.

Reply:
Good catch, removed it https://github.com/WordPress/gutenberg/pull/57119/commits/7b94302b6ede080cec02a2b3d19611ad34ce44fb